src/main/java/nl/novi/techiteasy/controllers/TelevisionController.java

```diff
@@ -1,33 +1,64 @@
 package nl.novi.techiteasy.controllers;
 
+import nl.novi.techiteasy.dtos.TelevisionDto;
+import nl.novi.techiteasy.dtos.TelevisionInputDto;
+import nl.novi.techiteasy.services.TelevisionServiceImpl;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
 import org.springframework.web.bind.annotation.*;
 
+import javax.validation.Valid;
+import java.util.List;
+
 @RestController
 public class TelevisionController {
 
+    private final TelevisionServiceImpl televisionService;
+
+    public TelevisionController(TelevisionServiceImpl televisionService) {
+        this.televisionService = televisionService;
+    }
+
     @GetMapping("/televisions")
     public ResponseEntity<Object> getAllTelvisions() {
-        return ResponseEntity.ok();
+        List televisionList = televisionService.getTelevisions();
+        return ResponseEntity.ok().body(televisionList);
     }
 
     @GetMapping("/televisions/{id}")
     public ResponseEntity<Object> getTelvision(@PathVariable Long id) {
-        return ResponseEntity.ok();
+        TelevisionDto tv = televisionService.getTelevision(id);
+        return ResponseEntity.ok().body(tv);
     }
 
-    @PostMapping("/televisons")
-    public ResponseEntity<Object> addTelevision(@RequestBody String television) {
-        return ResponseEntity.created();
+    @PostMapping("/televisions")
+    public ResponseEntity<Object> addTelevision(@Valid @RequestBody TelevisionInputDto televisionInputDto, BindingResult br) {
```

Comment:
Heel god gebruik gemaakt van @Valid annotation. Jij hebt heev veel verschillende beperkingen van de gebruikers Input ingevuld in de TelevisionInputDto file - echt top!